worlds/kh2/Rules.py

```diff
@@ -83,7 +83,7 @@ def hundred_acre_unlocked(self, state: CollectionState, amount) -> bool:
         return state.has(ItemName.TornPages, self.player, amount)
 
     def level_locking_unlock(self, state: CollectionState, amount):
-        return amount <= sum([state.count(item_name, self.player) for item_name in visit_locking_dict["2VisitLocking"]])
+         return amount <= sum([state.count(item_name, self.player) for item_name in visit_locking_dict["2VisitLocking"]])
```

Comment:
As far as I can tell the indentation gets messed up here?

Reply:
yeah this is a fragment of when the ability dupe branch had the promise charm pr